README.md

```diff
@@ -157,6 +157,13 @@ roadster.wikipedia # 'https://en.wikipedia.org/wiki/Elon_Musk%27s_Tesla_Roadster
 roadster.details # "Elon Musk's Tesla Roadster is an electric sports car that served as the dummy payload for the February 2018 Falcon Heavy test flight and is now an artificial satellite of the Sun. Starman, a mannequin dressed in a spacesuit, occupies the driver's seat. The car and rocket are products of Tesla and SpaceX, both companies founded by Elon Musk. This 2008-model Roadster was previously used by Musk for commuting, and is the only consumer car sent into space."
 ```
 
+### Capsules
+```ruby
+capsule = SPACEX::Capsules.info
+
+roadster.info # ["dragon1", "dragon2"]
```

Comment:
Hi @zainalmustofa 

We should have something like: `capsule.info` instead of `roadster.info` here. 

Also, we're looking to retrieve the entire structure and not just the names. Like we have here: https://api.spacexdata.com/v3/capsules

```
{
    "capsule_serial": "C201",
    "capsule_id": "dragon2",
    "status": "active",
    "original_launch": null,
    "original_launch_unix": null,
    "missions": [],
    "landings": 0,
    "type": "Dragon 2.0",
    "details": "Pressure vessel used for Dragon 2 structural testing. Rumored to be repurposed for first Red Dragon Mission",
    "reuse_count": 0
}
```

Would you mind to add a test for the `Capsules` similar with what we have for the other methods?  https://github.com/rodolfobandeira/spacex/blob/master/spec/spacex/roadster_spec.rb#L10

Reply:
Okay, i will try, and i think we need update version API right?
from `v2 ` to `v3` https://github.com/rodolfobandeira/spacex/blob/master/lib/spacex/endpoint.rb ?

Reply:
Hey @zainalmustofa 
I don't think changing the API will affect the current data. At least I ran rspec under your branch and I didn't see anything failing. Thanks @zainalmustofa

Reply:
Yaa honestly i not create yet rspec for this one, will create soon

Reply:
Hi @rodolfobandeira any advice why my commit always fail at rubocop CI ?
for example 
```
spec/spacex/capsules_spec.rb:22:4: C: Final newline missing.
end
```
i not sure how to solve this one